### PR TITLE
#0: add op profile signpost support

### PR DESCRIPTION
**Pull request by nsmithtt**

### CODE REVIEW

**Review by mo-tenstorrent — CHANGES_REQUESTED** [review]
Let's hold off on this until Tracy based op reports are merged in. There might be python only options to achieve this then without the need for extra pybind functions.

**Comment by nsmithtt** [comment]
Holding off until Tracy based op reports are merged in.  Use python only function to achieve this

**Pull request closed without merging**
